kubeshop/monokle
# fix: fix glitch in git pane while remote URL is showing

This PR closes https://github.com/kubeshop/monokle/issues/4193

## Fixes

- Fixes a glitch where the changelist keeps moving when the remote URL input is showing.

## How to test it

- Open a git repository without remote origin
- Make changes in the file explorer
- Go to the git activity
- The changelist should no longer be moving

## Checklist
